.github/workflows/binaries.yml

```diff
@@ -32,11 +32,14 @@ jobs:
           mkdir $ZIPDIR
           if [ ${{matrix.name}} == "macos" ]
           then
+            # Let's ensure that both aarch64 and x86_64 targets are installed
+            rustup target add aarch64-apple-darwin x86_64-apple-darwin
```

Comment:
I don't love that only one of these two targets is definitely already installed but in the future this will be on an m1 so the x86_64 target will need to be installed.